controllers/schools.js

```diff
@@ -47,7 +47,7 @@ const getStorageTypes = () => [
 ];
 
 const getDateFormat = (date) => {
-  const formattedDate = moment(date).utc().format("DD.MM.YYYY HH:mm");
+  const formattedDate = moment(date).tz("Europe/Berlin").format("DD.MM.YYYY HH:mm");
```

Comment:
its set in env vars, simply dropping .utc() might solve the issue to?

see https://github.com/hpi-schul-cloud/superhero-dashboard/blob/d3584fa6dc255a2ad416061b8849d774f55eee58/Dockerfile#L3

Reply:
yes, you are right